nbstripout/_nbstripout.py

```diff
@@ -420,6 +442,11 @@ def main():
         'cell.metadata.hidden',
         'cell.metadata.scrolled',
     ]
+    if args.mode is None:
```

Comment:
This is no longer needed

Reply:
Good catch. I dont know how I missed this. Thanks a lot.